assets/backend/admin/components/DataViews/DataViewsFooter.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+	__experimentalHStack as HStack,
```

Comment:
For later: let's import all experimental components in the `experimental.ts` file, and import in our codebase from there.